morebits.js

```diff
@@ -2056,6 +2056,7 @@ Morebits.wiki.page = function(pageName, currentAction) {
 		watchlistOption: 'nochange',
 		creator: null,
 		timestamp: null,
+		canEdit: null,  // true or false after load
```

Comment:
It might be better for this to be a `testactions` array or something, then just assign all found actions to that array.  Then, separate getters can be used (`return ctx.testactions.indexOf('edit') !== -1`) or a bulk test (`function(action) { return ctx.testactions.indexOf(action) !== -1 }`)

Reply:
There are also concerns about initial value here.  Distinguishing `null` from `false` or `true` could be useful if there's a chance a page object hasn't been `load`ed yet...

Reply:
I've gone and done this, still just with `edit` for now.